managed-vpc/build.f/particles-initial/config/default.js

```diff
@@ -3,7 +3,7 @@ var config = {
     {
       aws: {
         region: 'us-east-1',
-        bucket: 'sgas.particles-managed-vpc.714270045944.us-east-1'
+        bucket: 'sgas.particles-managedvpc.714270045944.us-east-1'
```

Comment:
I tired , sgas.particles-managed-vpc.714270045944.us-east-1 this , I think Raj have used sgas.particles-**manage-vpc**.714270045944.us-east-1 , which is also not correct as its ManagedVPC